frontend/HelperFuncs/personalization.js

```diff
@@ -1,13 +1,11 @@
-// this file contains a suite of functions to assist in recomending goals on the basis of their transaction history relative to similar users
-
-
+// this file contains a suite of functions to assist in recommending goals on the basis of their transaction history relative to similar users
 
 /*
-the main function of this file. This function returns a set of goals my algorithm flags as being recomended a user pursue. 
+the main function of this file. This function returns a set of goals my algorithm flags as being recommended a user pursue. 
 for each sub category transaction in [transaction] for which the user has a non-null value my algorithm will find:
-If the user’s value is statistically significant (as measured by z-index)[4]
-If the user’s value is above the interquartile range[2]
-If a similar user completed a goal for this sub-category [1]
```

Comment:
curious if we could do both tracking and completed goals?

Reply:
I'll see how much time I have left today after I finish refactoring my project plan– if I have the bandwidth for it I agree this is a good natural progression